src/stream/src/executor/watermark_filter.rs

```diff
@@ -111,6 +112,11 @@ impl<S: StateStore> WatermarkFilterExecutor<S> {
             mut table,
         } = *self;
 
+        let eval_error_report = ActorEvalErrorReport {
+            actor_context: ctx.clone(),
+            identity: info.identity.into(),
+        };
```

Comment:
Since we are providing `identity` and `eval_error_report` etc in `params`, what about passing in `eval_error_report` when `new`?

Reply:
I agree that the fields here are somehow redundant. Either LGTM. 🥹